src/targets/target.js

```diff
@@ -53,6 +61,9 @@ export class Target extends ClickableObject {
     }
 
     clickTarget() {
+        // Click animation
```

Comment:
Right now we attack the trees with whatever is equipped. We should have a unique animation for that I'm guessing. Also might be good to only allow woodcutting after an axe has been purchased -> or just start with bronze as you do in rs tutorial.

Also looks like the axe animation hits the object with the wrong side of the axe. Honestly I don't care if that is how it is, but it does look a little weird and might not be how you intended

Reply:
Yeah I plan on doing a couple things relevant to that. 
Like make it so you can’t chop a tree w/o an axe. Do you think they should start with an axe or have to purchase one?
I also haven’t implemented level requirements yet so I’ll need to work on that, probably at the same time.

For the axe swing I tested a few items but not that one so maybe I’ll just mirror the image

Reply:
I'd say let them start with one

Reply:
Okay you start with an axe now and you can only chop down trees if an axe is equipped. Also I flipped the image if it's an axe so it looks right